spec/features/roles_spec.rb

```diff
@@ -98,6 +98,51 @@
     end
   end
 
+  context 'organization_admin' do
+    let!(:organization) { create(:organization) }
+    let!(:org_admin_role) { Role.find_by(name: 'organization_admin', resource: organization) }
+    let!(:organization_admin) { create(:user, role_ids: [org_admin_role.id]) }
+    let(:user_with_no_role) { create :user }
+    let!(:other_organization) { create(:organization) }
+
+    before do
+      sign_in organization_admin
+      visit admin_organizations_path
+    end
+
+    context 'for the organization it belongs to' do
+      scenario 'successfully adds role organization_admin' do
+        click_link('Admins', href: admins_admin_organization_path(organization.id))
+
+        fill_in 'user_email', with: user_with_no_role.email
+        click_button 'Add'
+        user_with_no_role.reload
+
+        expect(user_with_no_role.has_role?('organization_admin', organization)).to eq true
+      end
+
+      scenario 'successfully removes role organization_admin' do
+        click_link('Admins', href: admins_admin_organization_path(organization.id))
+
+        first('tr').find('.btn-danger').click
+        expect(organization_admin.has_role?('organization_admin', organization)).to eq false
+      end
+    end
+
+    context 'for the organizations it does not belong to' do
+      scenario 'does not successfully add role organization_admin' do
+        click_link('Admins', href: admins_admin_organization_path(other_organization.id))
+
+        expect(page.has_field?('user_email')).to eq false
+      end
+
+      scenario 'does not successfully removes role organization_admin' do
```

Comment:
These tests won't be needed if we only allow site admins or org admins of that particular organization to access `admins` action in organization_controller.  
  
There are two possible sane ways here:
- Allow any user with role to view organization admins of a organization
- Only allow site admins or org admins of a organization to view org admins of that organization.

If any user with role has access to view org admins then he/she will know who to contact in case of a query or problem related to conferences of that organization. So, I personally suggest the first one.
@lagartoflojo